app/config/index.js

```diff
@@ -155,4 +57,10 @@ if (error) {
   throw new Error(`The server config is invalid. ${error.message}`)
 }
 
+value.notify = notifyConfig
+value.applicationRequestQueue = messageQueueConfig.applicationRequestQueue
```

Comment:
To keep it tidy, you could just use:
`value.mqConfig= messageQueueConfig`

then when reference:
`const { fetchApplicationResponseMsgType, applicationResponseQueue } = require('../../config').mqConfig`

Reply:
Yes, makes sense.